README.md

```diff
@@ -2,24 +2,38 @@
 
 A minimal recommended setup for an applications using [Cesium](https://cesium.com) with [Webpack](https://webpack.js.org/concepts/).
 
-[![Build Status](https://travis-ci.org/CesiumGS/cesium-webpack-example.svg?branch=using-custom-loader)](https://travis-ci.org/CesiumGS/cesium-webpack-example)
+Jump to the [Webpack 5](./webpack-5/) directory for the most up to date example. We also provide a [Webpack 4](./webpack-4/) example if you are still on the older version.
 
 ## Running this application
 
 ```sh
+# switch to the corresponding webpack-4 or webpack-5 directory
 npm install
 npm start
+# for the built version
+npm run build
+npm run start:built
 ```
 
 Navigate to `localhost:8080`.
 
 ### Available scripts
 
-- `npm start` - Runs a webpack build with `webpack.config.js` and starts a development server
-- `npm run build` - Runs a webpack build with `webpack.config.js`
+The top level scripts in the root directory are mostly for development onf this repo itself:
+
+- `npm run eslint`, `npm run prettier`, `npm run prettier-check` - Lint this project to maintain code style
```

Comment:
This is only targeted for developers, right? Maybe this should be moved under the `Contributions` section.